test/cypress/integration/group-chat-direct-message.spec.js

```diff
@@ -199,6 +199,9 @@ describe('Create/Join direct messages and orders of direct message channels', ()
 
     openDMByMembers([user3])
 
+    // This is a fix for a heisenbug: https://github.com/okTurtles/group-income/issues/2931
+    cy.wait(4 * 1000) // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
tbh, I don't have a clear reason why it fixes the heisen-bug.
A while ago, I remember having a chat with either Ricardo or Alex where he mentioned the connection in the CI environment might be unstable and that sometimes makes operations stack up which could lead to this kind of heisen-bug. So we sometimes need to manually inject some wait time between some heavy operations.

Thought this might be the case here, so added this logic here.